spec/javascript/components/IndexTable.test.js

```diff
@@ -22,6 +28,15 @@ describe('IndexIndexTable', () => {
     expect(tree).toMatchSnapshot()
   })
 
+  test('Should render Empty string if values are undefined', () => {
```

Comment:
I added this test to check if errors are raised if undefined is passed into the table cell. I was able to confirm that if i remove my check if val is undefined then the tests fail